packages/early-boot-config/early-boot-config.spec

```
@@ -0,0 +1,152 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+Name: %{_cross_os}early-boot-config
+Version: 0.0
+Release: 0%{?dist}
+Summary: early-boot-config
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source100: early-boot-config.service
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package -n %{_cross_os}early-boot-config-common
+Summary: early-boot-config binary
+%description -n %{_cross_os}early-boot-config-common
+%{summary}.
+
+%package -n %{_cross_os}early-boot-config-data-providers
+Summary: early-boot-config data providers directory
+%description -n %{_cross_os}early-boot-config-data-providers
+%{summary}.
+
+%package -n %{_cross_os}early-boot-config-local
+Summary: local-provider
+Requires: %{_cross_os}early-boot-config-data-providers
+%description -n %{_cross_os}early-boot-config-local
+%{summary}.
+
+%package -n %{_cross_os}early-boot-config-aws
+Summary: early-boot-config package for AWS
+Provides: %{_cross_os}early-boot-config
+Requires: %{_cross_os}early-boot-config-common
```

If you take my suggestion to factor the "common" subpackage back into the top-level one:
```suggestion
Requires: %{name}
```